src/spyglass/spikesorting/v1/utils.py

```diff
@@ -62,29 +71,22 @@ def get_spiking_sorting_v1_merge_ids(restriction: dict):
     ]
     # list of sorting ids for each recording
     sorting_restriction = restriction.copy()
-    del sorting_restriction["interval_list_name"]
+    _ = sorting_restriction.pop("interval_list_name", None)
```

Comment:
What's the reasoning for preferring `pop` to `del`?

Reply:
`pop` prevents an error in the event the key doesn't exist